plugins/catalog-backend/src/catalog/types.ts

```diff
@@ -311,3 +311,29 @@ export type Cursor = {
    */
   totalItems?: number;
 };
+
+/**
+ * The details of the conflicting entity.
+ */
+export type CatalogConflictEventPayload = {
+  /**
+   * The new conflicting entity
+   */
+  entity: Entity;
```

Comment:
This field is a bit interesting, and it might not have the semantics that people think it has. It'll be unprocessed and unvalidated; as a matter of fact, it might be so broken that it doesn't even abide by the basic Entity schema. If I recall correctly, all we do upfront is to check that kind and metadata.name exist and are strings, basically.

Could be worth calling that out in the comment at least.

Reply:
Good point on this - does that change I pushed work?

Reply:
What about calling it `unprocessedEntity`?

Reply:
Yes good idea